README.md

```diff
@@ -1,2 +1,40 @@
 # Prank-amos
 List of pranks that you can do to your office mates 😁
+
```

Comment:
Soo.... we need to define a structure for the list of pranks.

Talking with @portellaa we were thinking that the main `README.md` could have the list of pranks and then each prank would have its folder with the `README.md` there.

What do you think?

Reply:
Moved the prank to it's own file and added the link in this page

Reply:
Let's discuss this 🤔 

perhaps with links, placing this files inside folders

Reply:
![](https://www.ultimate-guitar.com/static/article/news/7/75897_0_wide_ver1531632261.jpg@642)